src/sentry/conf/server.py

```diff
@@ -301,6 +301,7 @@ def env(
 # response modifying middleware reset the Content-Length header.
 # This is because CommonMiddleware Sets the Content-Length header for non-streaming responses.
 MIDDLEWARE = (
+    "csp.middleware.CSPMiddleware",
```

Comment:
Have you done any testing on how this will interact with the CSP header middleware we have in getsentry? Having one way to do CSP in both would be nice.

Reply:
Yeah, I am planning to swap custom implementation with django-csp in getsentry.